packages/eslint-plugin-wantedly/docs/rules/no-data-testid.md

```diff
@@ -0,0 +1,45 @@
+# Check if disallowed prop does not appeared in jsx
+
+## Rule Details
+
+#### Valid
+
+```jsx
+const Component = () => {
+  return (
+    <div>
+      Content
+    </div>
+  );
+}
+```
+
+#### Invalid
+
+```jsx
+const Component = () => {
+  return (
+    <div data-testid="foo">     // <- data-testid must not be set as a prop
+      Content
+    </div>
+  );
+}
+```
+
+## Options
+
+#### `denyKeyList`
+
+The list of name which is not allowed to assign as prop. Default is `["data-testid"]`.
+
+```json
+{
+  "rule": {
+    "wantedly/no-data-testid": ["error": { "denyKeyList": ["data-testid", "data-test-id"] }]
```

Comment:
I think the `denyKeyList` is not only aim to deny the `data-testid`. If the user specify the `key` in this rule option, the user prohibit to set the key in the JSX Element. So I don't know the rule name `wantedly/no-data-testid` is appropriate.